src/views/Exchange/Exchange.js

```diff
@@ -173,10 +174,23 @@ export function getPositionQuery(tokens, nativeTokenAddress) {
 }
 
 export default function Exchange({ savedIsPnlInLeverage, setSavedIsPnlInLeverage, savedSlippageAmount, pendingTxns, setPendingTxns, savedShouldShowPositionLines, setSavedShouldShowPositionLines }) {
+  const [bannerHidden, setBannerHidden] = useLocalStorageSerializeKey('bannerHidden', null)
+
+  const hideBanner = (e) => {
+    const hiddenLimit = new Date(new Date().getTime()+(2*24*60*60*1000));
+    setBannerHidden(hiddenLimit)
+  }
+
   useEffect(() => {
     window.scrollTo(0, 0)
   }, [])
 
+  useEffect(() => {
+    if (bannerHidden && new Date(bannerHidden) <= new Date() && new Date() > new Date('2021-11-30')) {
```

Comment:
i guess there is a bug: if date > 2021-11-30 (= trading competition is ended) we should not show banner at all. and with current logic it will be shown. 

please test all cases to ensure everything works fine